### PR TITLE
Add cipher tests

### PR DESCRIPTION
**Pull request by AurelienFT**

### CODE REVIEW

File: massa-cipher/src/tests/mod.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    fn test_encrypt() {
+        let password = "password";
+        let data = "data";
+
+        let result = encrypt(password, data.as_bytes());
+        assert!(result.is_ok());
+
+        let cipher_data = result.unwrap();
+        assert_eq!(cipher_data.encrypted_bytes.len(), 20);
+        assert_eq!(cipher_data.salt.len(), 16);
```
Comment (sydhds) [inline massa-cipher/src/tests/mod.rs:16]: constant: SALT_SIZE here?

File: massa-cipher/src/tests/mod.rs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        assert_eq!(cipher_data.nonce.len(), 12);
```
Comment (sydhds) [inline massa-cipher/src/tests/mod.rs:17]: constant: NONCE_SIZE here?

File: massa-cipher/src/tests/mod.rs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    #[test]
+    fn test_encrypt() {
+        let password = "password";
+        let data = "data";
+
+        let result = encrypt(password, data.as_bytes());
+        assert!(result.is_ok());
+
+        let cipher_data = result.unwrap();
+        assert_eq!(cipher_data.encrypted_bytes.len(), 20);
```
Comment (sydhds) [inline massa-cipher/src/tests/mod.rs:15]: use HASH_PARAMS - NONCE_SIZE here?

**Review by sydhds — CHANGES_REQUESTED** [review]

File: massa-cipher/src/tests/mod.rs (hunk 1 above)
Comment (AurelienFT) [reply massa-cipher/src/tests/mod.rs]: Wasn't aware of these values. Thank you. Fixed

File: massa-cipher/src/tests/mod.rs (hunk 2 above)
Comment (AurelienFT) [reply massa-cipher/src/tests/mod.rs]: Fixed

**Review by sydhds — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**